modules/check_in/app/serializers/check_in/v2/appointment_data_serializer.rb

```diff
@@ -53,12 +53,12 @@ class AppointmentDataSerializer
           emergencyContactNeedsUpdate: raw_confirmation[:emergencyContactNeedsUpdate],
           emergencyContactConfirmedAt: raw_confirmation[:emergencyContactConfirmedAt]
         }
-        set_e_check_in_called = object.payload[:setECheckinStartedCalled]
+        set_e_check_in_started_called = object.payload[:setECheckinStartedCalled]
```

Comment:
would recommend keeping the casing and camelcase conversion consistent with what's coming back from CHIP:
```
        set_e_checkin_started_called = object.payload[:setECheckinStartedCalled]
```

Reply:
Here, `set_e_checkin_started_called` is a temporary variable like `raw_confirmation` or `emergency_contact` used above.  Below code block is the actual hash passed to front end and uses camel case

Reply:
Thanks for clarifying.  Thats a good catch.  Fixed in next commit